genomics_quality_control/genomics_quality_control.md

```diff
@@ -0,0 +1,927 @@
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For more information on why this particular dataset was chosen, see the [Data Carpentry summary page on these data](https://datacarpentry.org/organization-genomics/data/).
+
+</div>
+
+### This dataset
+
+The data we are going to use is part of a long-term evolution experiment led by [Richard Lenski](https://en.wikipedia.org/wiki/E._coli_long-term_evolution_experiment).
+
+The experiment was designed to assess adaptation in *E. coli*. A population was propagated for more than 40,000 generations in a glucose-limited minimal medium (in most conditions glucose is the best carbon source for *E. coli*, providing faster growth than other sugars). This medium was supplemented with citrate, which *E. coli* cannot metabolize in the aerobic conditions of the experiment. 
+
+Sequencing of the populations at regular time points revealed that spontaneous citrate-using variant (**Cit+**) appeared between 31,000 and 31,500 generations, causing an increase in population size and diversity. In addition, this experiment showed hypermutability in certain regions. Hypermutability is important and can help accelerate adaptation to novel environments, but also can be selected against in well-adapted populations.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To read more about the history of the experiment, see this paper [Blount et al. 2008: Historical contingency and the evolution of a key innovation in an experimental population of *Escherichia coli*](http://www.pnas.org/content/105/23/7899).
+
+A timeline of the experiment from Wikipedia shows a history of major events (the timeline only shows to 2016, but note that the experiment is on-going): 
+
+![*E. coli* long-term evolution experiment timeline from 1988 to 2016, with *E. coli* generations from 0 to 65,000. The **Cit+** variant does not appear until approximately 31,000 generations, more than a decade into the experiment.](media/LTEE_Timeline.png "Credit: [Zachary Blount](https://commons.wikimedia.org/wiki/User:Uncle_Zooey), [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0).")<!-- style = "max-width:300px; display: block; margin-left: auto;margin-right: auto;" -->
+
+</div>
+
+### The metadata
+
+We will be working with three sample events from the **Ara-3** strain of this experiment, one from 5,000 generations, one from 15,000 generations, and one from 50,000 generations.
+The population changed substantially during the course of the experiment, and we will be exploring how (the evolution of a **Cit+** mutant and **hypermutability**) with our variant calling workflow.
+
+The metadata file associated with this lesson can be [downloaded directly here](files/Ecoli_metadata_composite.csv) or [viewed in Github](https://github.com/datacarpentry/wrangling-genomics/blob/main/episodes/files/Ecoli_metadata_composite.csv). 
```

Comment:
It doesn't look like the CSV file has been added to our repo yet, so the first link is broken.

Reply:
Ah good catch! Thank you!